src/NCalc.Sync/ExpressionFunctionData.cs

```diff
@@ -0,0 +1,14 @@
+namespace NCalc;
+
+public class ExpressionFunctionData(Guid id, Expression[] arguments, ExpressionContext context)
```

Comment:
I don't see any way to get the number of arguments, since `Arguments` is private.